[0m2021.08.24 01:56:42 INFO  Started: Metals version 0.10.5 in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle' for client Visual Studio Code 1.59.1.[0m
[0m2021.08.24 01:56:43 INFO  time: initialize in 1.41s[0m
[0m2021.08.24 01:56:43 WARN  Build server is not auto-connectable.[0m
[0m2021.08.24 01:56:43 WARN  no build tool detected in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. [0m
[0m2021.08.24 01:56:43 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp1/Exp1.scala[0m
[0m2021.08.24 01:56:46 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.08.24 01:56:48 INFO  time: code lens generation in 4.08s[0m
8Êúà 24, 2021 2:05:22 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 5
[0m2021.08.24 02:05:22 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Main.scala[0m
[0m2021.08.24 02:06:30 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/test.scala[0m
[0m2021.08.24 02:06:58 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Main.scala[0m
[0m2021.08.24 02:07:14 ERROR text document: file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Main.scala
java.nio.file.NoSuchFileException: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Main.scala
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219)
	at java.nio.file.Files.newByteChannel(Files.java:371)
	at java.nio.file.Files.newByteChannel(Files.java:422)
	at java.nio.file.Files.readAllBytes(Files.java:3206)
	at scala.meta.internal.io.PlatformFileIO$.slurp(PlatformFileIO.scala:45)
	at scala.meta.internal.io.FileIO$.slurp(FileIO.scala:24)
	at scala.meta.internal.metals.InteractiveSemanticdbs.$anonfun$textDocument$1(InteractiveSemanticdbs.scala:77)
	at java.util.HashMap.compute(HashMap.java:1228)
	at java.util.Collections$SynchronizedMap.compute(Collections.java:2692)
	at scala.meta.internal.metals.InteractiveSemanticdbs.textDocument(InteractiveSemanticdbs.scala:76)
	at scala.meta.internal.metals.AggregateSemanticdbs.loop$1(AggregateSemanticdbs.scala:30)
	at scala.meta.internal.metals.AggregateSemanticdbs.textDocument(AggregateSemanticdbs.scala:36)
	at scala.meta.internal.metals.CodeLensProvider.findLenses(CodeLensProvider.scala:22)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeLens$2(MetalsLanguageServer.scala:1507)
	at scala.meta.internal.metals.TimerProvider.timedThunk(TimerProvider.scala:25)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeLens$1(MetalsLanguageServer.scala:1505)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$apply$2(CancelTokens.scala:26)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.lang.Thread.run(Thread.java:829)
[0m
[0m2021.08.24 02:07:16 ERROR text document: file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Main.scala
java.nio.file.NoSuchFileException: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Main.scala
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219)
	at java.nio.file.Files.newByteChannel(Files.java:371)
	at java.nio.file.Files.newByteChannel(Files.java:422)
	at java.nio.file.Files.readAllBytes(Files.java:3206)
	at scala.meta.internal.io.PlatformFileIO$.slurp(PlatformFileIO.scala:45)
	at scala.meta.internal.io.FileIO$.slurp(FileIO.scala:24)
	at scala.meta.internal.metals.InteractiveSemanticdbs.$anonfun$textDocument$1(InteractiveSemanticdbs.scala:77)
	at java.util.HashMap.compute(HashMap.java:1228)
	at java.util.Collections$SynchronizedMap.compute(Collections.java:2692)
	at scala.meta.internal.metals.InteractiveSemanticdbs.textDocument(InteractiveSemanticdbs.scala:76)
	at scala.meta.internal.metals.AggregateSemanticdbs.loop$1(AggregateSemanticdbs.scala:30)
	at scala.meta.internal.metals.AggregateSemanticdbs.textDocument(AggregateSemanticdbs.scala:36)
	at scala.meta.internal.metals.CodeLensProvider.findLenses(CodeLensProvider.scala:22)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeLens$2(MetalsLanguageServer.scala:1507)
	at scala.meta.internal.metals.TimerProvider.timedThunk(TimerProvider.scala:25)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$codeLens$1(MetalsLanguageServer.scala:1505)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$apply$2(CancelTokens.scala:26)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.lang.Thread.run(Thread.java:829)
[0m
[0m2021.08.24 02:08:41 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala[0m
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala, 35, 35, 50)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala, 35, 35, 43)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala in Array[String]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala, 35, 35, 48)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala in Array[String]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala, 35, 35, 48)
[0m2021.08.24 02:21:53 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala[0m
[0m2021.08.24 02:21:53 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala[0m
[0m2021.08.24 02:21:53 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala[0m
[0m2021.08.24 02:24:04 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala[0m
[0m2021.08.24 02:24:58 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala[0m
[0m2021.08.24 02:25:20 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala[0m
[0m2021.08.24 02:25:20 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala[0m
[0m2021.08.24 02:25:20 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala[0m
[0m2021.08.24 02:25:28 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala[0m
8Êúà 24, 2021 2:25:29 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2083
8Êúà 24, 2021 2:25:32 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2092
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala, 78, 78, 93)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala, 78, 78, 94)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala, 78, 78, 87)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala, 78, 78, 88)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala in Array[Int]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala, 78, 78, 89)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala in Array[Int]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala, 78, 78, 90)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala in Array[Int]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala, 78, 78, 90)
[0m2021.08.24 02:36:41 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala[0m
[0m2021.08.24 02:36:41 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala[0m
[0m2021.08.24 02:36:41 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala[0m
[0m2021.08.24 02:37:20 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala[0m
[0m2021.08.24 02:37:20 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Advance.scala[0m
[0m2021.08.24 02:38:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/AdvanceMemo.scala[0m
[0m2021.08.24 02:39:04 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/AdvanceLoop.scala[0m
8Êúà 24, 2021 2:39:05 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2918
8Êúà 24, 2021 2:39:22 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2949
8Êúà 24, 2021 2:39:23 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 16 is not a valid line number, allowed [0..0]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 16 is not a valid line number, allowed [0..0]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 16 is not a valid line number, allowed [0..0]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 24, 2021 2:39:28 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 16 is not a valid line number, allowed [0..0]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 16 is not a valid line number, allowed [0..0]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 16 is not a valid line number, allowed [0..0]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 24, 2021 2:44:19 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3601
8Êúà 24, 2021 2:47:56 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4368
8Êúà 24, 2021 2:48:01 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4377
8Êúà 24, 2021 2:48:02 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4383
8Êúà 24, 2021 2:48:09 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4403
8Êúà 24, 2021 2:48:14 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4413
[0m2021.08.24 02:48:32 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/AdvanceLoop.scala[0m
[0m2021.08.24 02:48:32 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/AdvanceLoop.scala[0m
[0m2021.08.26 01:35:39 INFO  Started: Metals version 0.10.5 in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle' for client Visual Studio Code 1.59.1.[0m
[0m2021.08.26 01:35:41 INFO  time: initialize in 1.38s[0m
[0m2021.08.26 01:35:41 WARN  Build server is not auto-connectable.[0m
[0m2021.08.26 01:35:41 WARN  no build tool detected in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. [0m
[0m2021.08.26 01:35:44 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/permutation.scala[0m
[0m2021.08.26 01:35:46 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.08.26 01:35:47 INFO  time: code lens generation in 3.27s[0m
[0m2021.08.26 01:36:48 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala, 41, 41, 56)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala, 41, 41, 49)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala, 41, 41, 50)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala in Array[String]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala, 41, 41, 54)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala in Array[String]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala, 41, 41, 54)
[0m2021.08.26 01:38:13 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/AdvanceLoop.scala[0m
8Êúà 26, 2021 1:38:19 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 454
[0m2021.08.26 01:38:20 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/permutation.scala[0m
[0m2021.08.26 01:38:58 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/permutation.scala[0m
[0m2021.08.26 01:38:58 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/permutation.scala[0m
8Êúà 26, 2021 1:38:58 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 541
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala, 82, 82, 97)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala, 82, 82, 90)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala in Array[Int]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala, 82, 82, 92)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala in Array[Int]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala, 82, 82, 92)
[0m2021.08.26 01:55:27 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:55:27 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:56:06 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/AdvanceLoop.scala[0m
8Êúà 26, 2021 1:56:15 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2526
[0m2021.08.26 01:56:21 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/AdvanceMemo.scala[0m
[0m2021.08.26 01:56:25 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/Basic.scala[0m
[0m2021.08.26 01:56:25 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/AdvanceMemo.scala[0m
8Êúà 26, 2021 1:56:27 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2562
8Êúà 26, 2021 1:56:28 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2567
[0m2021.08.26 01:56:28 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:56:28 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:56:28 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:57:14 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:57:14 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:57:14 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:57:39 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:57:39 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:57:39 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:57:59 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:57:59 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:57:59 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:58:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:58:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:58:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:58:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:58:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:58:32 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:58:32 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:58:32 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 01:59:07 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/Combination.scala[0m
[0m2021.08.26 02:22:32 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp1/Exp1.scala[0m
[0m2021.08.26 02:22:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/AdvanceLoop.scala[0m
[0m2021.08.26 02:22:46 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/CountDivideNum/AdvanceLoop.scala[0m
8Êúà 26, 2021 2:22:55 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2743
[0m2021.08.26 02:23:07 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala, 257, 257, 272)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala, 257, 257, 265)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala, 257, 257, 266)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala in Array[String]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala, 257, 257, 270)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala in Array[String]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala, 257, 257, 270)
[0m2021.08.26 02:26:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
[0m2021.08.26 02:26:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
[0m2021.08.26 02:26:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
[0m2021.08.26 02:26:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
[0m2021.08.26 02:26:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
[0m2021.08.26 02:26:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
[0m2021.08.26 02:26:18 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
8Êúà 26, 2021 2:26:21 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2959
8Êúà 26, 2021 2:26:23 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2964
8Êúà 26, 2021 2:26:27 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2972
8Êúà 26, 2021 2:32:12 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3658
8Êúà 26, 2021 2:32:17 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3667
8Êúà 26, 2021 2:32:20 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3677
[0m2021.08.26 02:32:30 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
8Êúà 26, 2021 2:32:31 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3698
[0m2021.08.26 02:35:40 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp1/Exp1.scala[0m
[0m2021.08.26 02:35:40 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp1/Exp1.scala[0m
[0m2021.08.26 02:35:40 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp1/Exp1.scala[0m
[0m2021.08.26 02:36:15 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp1/Exp1.scala[0m
[0m2021.08.26 02:36:15 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp1/Exp1.scala[0m
[0m2021.08.26 02:51:22 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
8Êúà 26, 2021 2:51:23 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4428
[0m2021.08.26 02:51:34 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 02:57:39 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 02:57:39 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 02:58:11 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 02:58:11 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 02:58:37 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 02:58:37 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 03:01:05 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 03:01:05 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 03:01:05 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 03:01:58 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 03:03:04 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 03:03:04 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 03:03:04 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.26 03:14:28 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
8Êúà 26, 2021 3:14:33 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 5795
8Êúà 26, 2021 3:14:38 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 5803
8Êúà 26, 2021 4:13:59 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 8631
[0m2021.08.26 04:15:08 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.26 04:15:08 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.26 04:19:09 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.26 04:19:09 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.26 04:19:09 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.26 04:19:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.26 04:19:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.26 04:21:25 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.26 04:22:51 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.26 04:22:52 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.26 04:24:48 INFO  shutting down Metals[0m
[0m2021.08.28 00:00:06 INFO  Started: Metals version 0.10.5 in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle' for client Visual Studio Code 1.59.1.[0m
[0m2021.08.28 00:00:08 INFO  time: initialize in 1.4s[0m
[0m2021.08.28 00:00:08 WARN  Build server is not auto-connectable.[0m
[0m2021.08.28 00:00:08 WARN  no build tool detected in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. [0m
[0m2021.08.28 00:00:08 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:00:11 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.08.28 00:00:13 INFO  time: code lens generation in 1.38s[0m
[0m2021.08.28 00:00:13 INFO  time: code lens generation in 2.23s[0m
[0m2021.08.28 00:00:13 INFO  time: code lens generation in 3.87s[0m
[0m2021.08.28 00:00:13 INFO  time: code lens generation in 4.62s[0m
8Êúà 28, 2021 12:00:14 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object Exp3
java.util.concurrent.CompletionException: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object Exp3
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:314)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:319)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:42)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object Exp3

[0m2021.08.28 00:03:43 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:03:43 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:03:43 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:04:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:04:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:04:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:14:46 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:16:26 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:20:03 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:20:03 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:20:19 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:28:38 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
8Êúà 28, 2021 12:29:43 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2073
[0m2021.08.28 00:29:49 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:29:49 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:29:49 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:30:12 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:30:12 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:36:07 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:36:12 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:36:14 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:36:14 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.28 00:36:18 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp4/Exp4.scala[0m
[0m2021.08.28 00:36:22 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
8Êúà 28, 2021 12:36:27 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2186
8Êúà 28, 2021 12:36:40 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2210
[0m2021.08.28 00:36:47 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
[0m2021.08.28 00:37:00 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
[0m2021.08.28 00:37:06 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
8Êúà 28, 2021 12:37:10 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2293
8Êúà 28, 2021 12:40:25 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 25 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 25 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 25 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:40:25 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 25 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 25 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 25 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:41:11 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:41:12 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:41:13 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:41:13 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:41:14 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:41:16 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:41:18 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:41:18 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:41:19 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:41:19 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:41:20 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:41:22 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 28, 2021 12:41:23 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 26 is not a valid line number, allowed [0..24]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

Exception in thread "pool-3-thread-1" java.lang.InterruptedException
	at scala.meta.internal.metals.FutureCancelToken.checkCanceled(FutureCancelToken.scala:29)
	at scala.meta.internal.pc.CompilerAccess.$anonfun$onCompilerJobQueue$1(CompilerAccess.scala:195)
	at scala.meta.internal.pc.CompilerJobQueue$Job.run(CompilerJobQueue.scala:139)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
[0m2021.08.28 00:52:39 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp4/Exp4.scala[0m
[0m2021.08.28 00:52:39 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp4/Exp4.scala[0m
[0m2021.08.28 00:52:55 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp4/Exp4.scala[0m
[0m2021.08.28 00:52:55 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp4/Exp4.scala[0m
[0m2021.08.28 00:53:29 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp4/Exp4.scala[0m
[0m2021.08.28 00:53:48 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp4/Exp4.scala[0m
[0m2021.08.28 00:54:10 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp4/Exp4.scala[0m
[0m2021.08.28 01:26:09 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
8Êúà 28, 2021 1:26:14 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 5424
8Êúà 28, 2021 1:26:19 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 5435
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala, 490, 490, 505)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala, 490, 490, 498)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala, 490, 490, 499)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala, 490, 490, 501)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala, 490, 490, 502)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala, 496, 496, 511)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala, 496, 496, 504)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala, 496, 496, 505)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala in Array[Int]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala, 496, 496, 506)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala in Array[Int]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala, 496, 496, 506)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala in Array[Int]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala, 496, 496, 506)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala in Array[Array[Int]]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala, 490, 490, 507)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala in Array[Int]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala, 516, 516, 526)
[0m2021.08.28 01:56:23 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 01:56:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 01:56:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 01:57:01 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 01:57:24 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 01:57:33 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 01:57:33 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 01:57:53 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:00:04 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:00:30 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:00:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:01:30 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:01:30 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:01:49 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:02:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
8Êúà 28, 2021 2:06:12 ÂçàÂâç scala.meta.internal.pc.CompilerAccess retryWithCleanCompiler
ÊÉÖÂ†±: compiler crashed due to an error in the Scala compiler, retrying with new compiler instance.
8Êúà 28, 2021 2:06:12 ÂçàÂâç scala.meta.internal.pc.CompilerAccess handleError
ÈáçÂ§ß: null
java.lang.NullPointerException
	at scala.reflect.internal.Definitions$DefinitionsClass.isByNameParamType(Definitions.scala:417)
	at scala.reflect.internal.TreeInfo.isStableIdent(TreeInfo.scala:140)
	at scala.reflect.internal.TreeInfo.isStableIdentifier(TreeInfo.scala:113)
	at scala.reflect.internal.TreeInfo.isPath(TreeInfo.scala:102)
	at scala.reflect.internal.TreeInfo.admitsTypeSelection(TreeInfo.scala:158)
	at scala.tools.nsc.interactive.Global.stabilizedType(Global.scala:960)
	at scala.tools.nsc.interactive.Global.typedTreeAt(Global.scala:808)
	at scala.meta.internal.pc.SignatureHelpProvider.signatureHelp(SignatureHelpProvider.scala:23)

[0m2021.08.28 02:11:56 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:13:15 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:13:38 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:14:21 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:14:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:15:06 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:15:06 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:15:38 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:16:13 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:16:23 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:19:32 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:20:02 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:21:06 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:21:06 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:21:06 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:21:06 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:21:06 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:21:24 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:21:24 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:21:52 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 02:21:52 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.28 05:32:49 INFO  shutting down Metals[0m
[0m2021.08.29 01:42:59 INFO  Started: Metals version 0.10.5 in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle' for client Visual Studio Code 1.59.1.[0m
[0m2021.08.29 01:43:01 INFO  time: initialize in 1.46s[0m
[0m2021.08.29 01:43:01 WARN  Build server is not auto-connectable.[0m
[0m2021.08.29 01:43:01 WARN  no build tool detected in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. [0m
[0m2021.08.29 01:43:01 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.29 01:43:04 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.08.29 01:43:05 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala[0m
[0m2021.08.29 01:43:05 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
[0m2021.08.29 01:43:06 INFO  time: code lens generation in 4.5s[0m
8Êúà 29, 2021 1:43:10 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 19
8Êúà 29, 2021 1:45:08 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 35
[0m2021.08.29 01:45:41 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp1/Exp1.scala[0m
[0m2021.08.29 01:46:16 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp1/Exp1.scala[0m
[0m2021.08.29 01:46:19 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.29 01:46:43 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp2/Exp2.scala[0m
[0m2021.08.29 01:46:46 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
8Êúà 29, 2021 1:46:52 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 262
[0m2021.08.29 01:48:29 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp3/Exp3.scala[0m
[0m2021.08.29 01:48:31 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp4/Exp4.scala[0m
[0m2021.08.29 01:48:48 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp4/Exp4.scala[0m
8Êúà 29, 2021 1:48:51 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 557
[0m2021.08.29 01:49:15 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.29 01:49:15 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.29 01:49:15 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
8Êúà 29, 2021 1:49:57 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 710
8Êúà 29, 2021 1:49:58 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 716
8Êúà 29, 2021 1:50:02 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 727
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala in Array[String]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala, 238, 238, 251)
[0m2021.08.29 01:57:11 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala[0m
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala, 60, 60, 75)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala, 60, 60, 68)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala in Array[Int]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala, 60, 60, 70)
[0m2021.08.29 02:01:14 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala[0m
[0m2021.08.29 02:01:14 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala[0m
[0m2021.08.29 02:01:14 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala[0m
[0m2021.08.29 02:02:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala[0m
[0m2021.08.29 02:03:15 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala[0m
[0m2021.08.29 02:03:47 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala[0m
[0m2021.08.29 02:03:47 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala[0m
[0m2021.08.29 02:03:47 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp6/Exp6.scala[0m
[0m2021.08.29 02:09:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
8Êúà 29, 2021 2:12:04 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3249
8Êúà 29, 2021 2:12:07 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3255
[0m2021.08.29 02:12:09 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
8Êúà 29, 2021 2:12:13 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3267
[0m2021.08.29 02:12:13 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/permutation.scala[0m
8Êúà 29, 2021 2:12:21 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3274
[0m2021.08.29 02:12:25 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
[0m2021.08.29 02:12:26 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Template.scala[0m
8Êúà 29, 2021 2:12:26 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3286
8Êúà 29, 2021 2:12:30 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3296
8Êúà 29, 2021 2:12:37 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3308
8Êúà 29, 2021 2:12:45 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3322
[0m2021.08.29 02:12:48 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp5/Exp5.scala[0m
[0m2021.08.29 02:13:29 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/permutation.scala[0m
8Êúà 29, 2021 2:13:36 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 3437
[0m2021.08.29 02:16:04 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
[0m2021.08.29 02:16:04 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
[0m2021.08.29 02:16:40 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
[0m2021.08.29 02:17:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
8Êúà 29, 2021 2:18:12 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 29, 2021 2:18:14 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 29, 2021 2:18:15 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 29, 2021 2:18:15 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 29, 2021 2:18:16 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 29, 2021 2:18:17 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 29, 2021 2:18:17 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 29, 2021 2:18:19 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

8Êúà 29, 2021 2:18:19 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 30 is not a valid line number, allowed [0..29]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

[0m2021.08.29 02:20:00 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
[0m2021.08.29 02:20:39 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
[0m2021.08.29 02:20:39 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
8Êúà 29, 2021 2:21:23 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4114
[0m2021.08.29 02:21:38 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
8Êúà 29, 2021 2:39:00 ÂçàÂâç scala.meta.internal.pc.CompilerAccess handleError
ÈáçÂ§ß: offset 416, count -1, length 742
java.lang.StringIndexOutOfBoundsException: offset 416, count -1, length 742
	at java.base/java.lang.String.checkBoundsOffCount(String.java:3304)
	at java.base/java.lang.String.rangeCheck(String.java:280)
	at java.base/java.lang.String.<init>(String.java:276)
	at scala.tools.nsc.interactive.Global.typeCompletions$1(Global.scala:1231)
	at scala.tools.nsc.interactive.Global.completionsAt(Global.scala:1254)
	at scala.meta.internal.pc.SignatureHelpProvider.$anonfun$treeSymbol$1(SignatureHelpProvider.scala:375)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.pc.SignatureHelpProvider.treeSymbol(SignatureHelpProvider.scala:373)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCall$.unapply(SignatureHelpProvider.scala:198)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCallTraverser.visit(SignatureHelpProvider.scala:309)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCallTraverser.traverse(SignatureHelpProvider.scala:303)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCallTraverser.fromTree(SignatureHelpProvider.scala:272)
	at scala.meta.internal.pc.SignatureHelpProvider.signatureHelp(SignatureHelpProvider.scala:27)

8Êúà 29, 2021 2:39:12 ÂçàÂâç scala.meta.internal.pc.CompilerAccess handleError
ÈáçÂ§ß: offset 416, count -1, length 759
java.lang.StringIndexOutOfBoundsException: offset 416, count -1, length 759
	at java.base/java.lang.String.checkBoundsOffCount(String.java:3304)
	at java.base/java.lang.String.rangeCheck(String.java:280)
	at java.base/java.lang.String.<init>(String.java:276)
	at scala.tools.nsc.interactive.Global.typeCompletions$1(Global.scala:1231)
	at scala.tools.nsc.interactive.Global.completionsAt(Global.scala:1254)
	at scala.meta.internal.pc.SignatureHelpProvider.$anonfun$treeSymbol$1(SignatureHelpProvider.scala:375)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.pc.SignatureHelpProvider.treeSymbol(SignatureHelpProvider.scala:373)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCall$.unapply(SignatureHelpProvider.scala:198)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCallTraverser.visit(SignatureHelpProvider.scala:309)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCallTraverser.traverse(SignatureHelpProvider.scala:303)
	at scala.meta.internal.pc.SignatureHelpProvider$MethodCallTraverser.fromTree(SignatureHelpProvider.scala:272)
	at scala.meta.internal.pc.SignatureHelpProvider.signatureHelp(SignatureHelpProvider.scala:27)

[0m2021.08.29 02:39:59 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
[0m2021.08.29 02:39:59 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
[0m2021.08.29 02:40:40 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
[0m2021.08.29 02:43:27 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp7/Exp7.scala[0m
8Êúà 29, 2021 2:49:44 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4328
[0m2021.08.29 02:50:07 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp8/Exp8.scala[0m
8Êúà 29, 2021 2:50:08 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4343
8Êúà 29, 2021 2:50:15 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4351
[0m2021.08.29 02:53:58 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp8/Exp8.scala[0m
[0m2021.08.29 03:09:23 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp8/Exp8.scala[0m
[0m2021.08.29 03:09:23 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp8/Exp8.scala[0m
[0m2021.08.29 03:10:56 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:10:58 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/permutation.scala[0m
8Êúà 29, 2021 3:10:59 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4730
[0m2021.08.29 03:11:04 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
8Êúà 29, 2021 3:11:14 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4752
8Êúà 29, 2021 3:11:17 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4759
8Êúà 29, 2021 3:18:14 ÂçàÂâç scala.meta.internal.pc.CompletionProvider expected$1
Ë≠¶Âëä: null
8Êúà 29, 2021 3:18:14 ÂçàÂâç scala.meta.internal.pc.CompletionProvider expected$1
Ë≠¶Âëä: null
8Êúà 29, 2021 3:18:14 ÂçàÂâç scala.meta.internal.pc.CompletionProvider expected$1
Ë≠¶Âëä: null
8Êúà 29, 2021 3:18:14 ÂçàÂâç scala.meta.internal.pc.CompletionProvider expected$1
Ë≠¶Âëä: null
8Êúà 29, 2021 3:18:14 ÂçàÂâç scala.meta.internal.pc.CompletionProvider expected$1
Ë≠¶Âëä: null
8Êúà 29, 2021 3:18:15 ÂçàÂâç scala.meta.internal.pc.CompletionProvider expected$1
Ë≠¶Âëä: null
8Êúà 29, 2021 3:18:15 ÂçàÂâç scala.meta.internal.pc.CompletionProvider expected$1
Ë≠¶Âëä: null
[0m2021.08.29 03:21:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:21:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:22:20 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:22:36 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:23:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:24:15 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:25:09 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:26:01 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:26:38 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
8Êúà 29, 2021 3:30:17 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 6979
[0m2021.08.29 03:30:39 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:32:00 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:44:34 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:44:43 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:45:10 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:45:10 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:45:10 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:45:10 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:45:11 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:46:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:46:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 03:46:19 INFO  shutting down Metals[0m
[0m2021.08.29 12:18:24 INFO  Started: Metals version 0.10.5 in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle' for client Visual Studio Code 1.59.1.[0m
[0m2021.08.29 12:18:26 INFO  time: initialize in 1.35s[0m
[0m2021.08.29 12:18:26 WARN  Build server is not auto-connectable.[0m
[0m2021.08.29 12:18:26 WARN  no build tool detected in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. [0m
[0m2021.08.29 12:18:26 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp9/Exp9.scala[0m
[0m2021.08.29 12:18:26 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
[0m2021.08.29 12:18:29 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.08.29 12:18:31 INFO  time: code lens generation in 4.4s[0m
[0m2021.08.29 12:20:24 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
8Êúà 29, 2021 12:20:30 ÂçàÂæå org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 36
8Êúà 29, 2021 12:20:34 ÂçàÂæå org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 42
8Êúà 29, 2021 12:20:39 ÂçàÂæå org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 51
[0m2021.08.29 12:20:41 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
8Êúà 29, 2021 12:20:42 ÂçàÂæå org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 61
8Êúà 29, 2021 12:20:43 ÂçàÂæå org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: java.lang.IllegalArgumentException: 14 is not a valid line number, allowed [0..3]
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: 14 is not a valid line number, allowed [0..3]
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:40)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalArgumentException: 14 is not a valid line number, allowed [0..3]
	at scala.meta.internal.inputs.InternalInput.lineToOffset(InternalInput.scala:37)
	at scala.meta.internal.inputs.InternalInput.lineToOffset$(InternalInput.scala:32)
	at scala.meta.inputs.Input$VirtualFile.lineToOffset(Input.scala:80)
	at scala.meta.inputs.Position$Range$.apply(Position.scala:59)
	at scala.meta.internal.mtags.MtagsEnrichments$XtensionPositionLspInverse.toMeta(MtagsEnrichments.scala:121)
	at scala.meta.internal.parsing.Trees.$anonfun$findLastEnclosingAt$5(Trees.scala:71)
	at scala.Option.flatMap(Option.scala:271)
	at scala.meta.internal.parsing.Trees.findLastEnclosingAt(Trees.scala:70)
	at scala.meta.internal.metals.codeactions.InsertInferredType.$anonfun$contribute$1(InsertInferredType.scala:70)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	... 4 more

[0m2021.08.29 12:20:55 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp8/Exp8.scala[0m
8Êúà 29, 2021 12:20:57 ÂçàÂæå org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 86
[0m2021.08.29 12:37:11 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 12:37:11 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 12:38:16 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 12:38:16 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 12:38:31 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 12:41:50 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala, 1050, 1050, 1065)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala, 1050, 1050, 1058)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala in Array[Int]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala, 1050, 1050, 1060)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala in Array[Int]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala, 1050, 1050, 1060)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala in Map[String,<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala, 1057, 1057, 1077)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala in Map[String,<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala, 1057, 1057, 1078)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala in Map[String,<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala, 1057, 1057, 1079)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala in Map[String,<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala, 1057, 1057, 1081)
[0m2021.08.29 13:00:33 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:00:33 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:00:33 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:01:13 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:01:13 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:01:43 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:01:43 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:03:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:03:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:08:56 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:10:06 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:10:06 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:11:18 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:11:18 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.29 13:12:10 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.30 00:07:43 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.30 00:07:43 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp10/Exp10.scala[0m
[0m2021.08.30 00:08:16 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
8Êúà 30, 2021 12:08:27 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4619
8Êúà 30, 2021 12:08:30 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4630
[0m2021.08.30 00:09:12 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/preface/permutation.scala[0m
8Êúà 30, 2021 12:09:16 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 4813
[0m2021.08.30 00:19:22 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
[0m2021.08.30 00:20:13 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
[0m2021.08.30 00:20:47 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
[0m2021.08.30 00:37:06 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
[0m2021.08.30 00:37:06 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
[0m2021.08.30 00:37:29 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
[0m2021.08.30 00:37:53 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
[0m2021.08.30 00:37:53 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
[0m2021.08.30 00:38:35 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
[0m2021.08.30 00:39:02 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
[0m2021.08.30 00:39:02 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
[0m2021.08.30 00:39:24 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
[0m2021.08.30 00:41:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp11/Exp11.scala[0m
[0m2021.08.30 00:50:22 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
8Êúà 30, 2021 1:02:14 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 7784
[0m2021.08.31 01:14:41 INFO  Started: Metals version 0.10.5 in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle' for client Visual Studio Code 1.59.1.[0m
[0m2021.08.31 01:14:42 INFO  time: initialize in 1.51s[0m
[0m2021.08.31 01:14:42 WARN  Build server is not auto-connectable.[0m
[0m2021.08.31 01:14:42 WARN  no build tool detected in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. [0m
[0m2021.08.31 01:14:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 01:14:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
[0m2021.08.31 01:14:45 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.08.31 01:14:47 INFO  time: code lens generation in 3.79s[0m
8Êúà 31, 2021 1:14:49 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 7
[0m2021.08.31 01:14:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
8Êúà 31, 2021 1:14:56 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 15
[0m2021.08.31 01:31:45 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 01:39:50 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 01:39:58 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 01:41:07 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 01:41:16 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 01:42:13 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 01:42:23 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 01:42:49 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
8Êúà 31, 2021 1:43:21 ÂçàÂâç scala.meta.internal.pc.CompilerAccess retryWithCleanCompiler
ÊÉÖÂ†±: compiler crashed due to an error in the Scala compiler, retrying with new compiler instance.
[0m2021.08.31 01:43:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
8Êúà 31, 2021 1:45:03 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 1185
[0m2021.08.31 01:46:34 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 01:46:54 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 01:47:26 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 01:58:50 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 01:58:59 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 01:59:17 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 02:01:36 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 02:01:38 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 02:02:02 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 02:02:20 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp12/Exp12.scala[0m
[0m2021.08.31 02:06:23 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.08.31 02:06:25 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/Template.scala[0m
8Êúà 31, 2021 2:06:30 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 2150
[0m2021.08.31 02:15:56 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.08.31 02:18:26 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.08.31 02:18:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
8Êúà 31, 2021 2:22:01 ÂçàÂâç scala.meta.internal.pc.CompilerAccess retryWithCleanCompiler
ÊÉÖÂ†±: compiler crashed due to an error in the Scala compiler, retrying with new compiler instance.
[0m2021.08.31 02:22:02 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.08.31 02:22:49 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.08.31 02:25:47 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.08.31 02:25:47 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.08.31 02:25:47 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.02 00:52:19 INFO  Started: Metals version 0.10.5 in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle' for client Visual Studio Code 1.59.1.[0m
[0m2021.09.02 00:52:21 INFO  time: initialize in 1.42s[0m
[0m2021.09.02 00:52:25 WARN  Build server is not auto-connectable.[0m
[0m2021.09.02 00:52:25 WARN  no build tool detected in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. [0m
[0m2021.09.02 00:52:25 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.02 00:52:27 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.09.02 00:52:30 INFO  time: code lens generation in 4.67s[0m
[0m2021.09.02 03:40:58 INFO  shutting down Metals[0m
[0m2021.09.09 01:58:10 INFO  Started: Metals version 0.10.5 in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle' for client Visual Studio Code 1.59.1.[0m
[0m2021.09.09 01:58:12 INFO  time: initialize in 1.47s[0m
[0m2021.09.09 01:58:12 WARN  Build server is not auto-connectable.[0m
[0m2021.09.09 01:58:12 WARN  no build tool detected in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. [0m
[0m2021.09.09 01:58:12 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 01:58:15 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
[0m2021.09.09 01:58:18 INFO  time: code lens generation in 4.98s[0m
9Êúà 09, 2021 2:08:44 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleCancellation
Ë≠¶Âëä: Unmatched cancel notification for request id 748
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala, 467, 467, 482)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala, 467, 467, 475)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala in Array[<error>]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala, 467, 467, 476)
9Êúà 09, 2021 2:12:02 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object Exp13
java.util.concurrent.CompletionException: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object Exp13
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:314)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:319)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:42)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object Exp13

something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala in Array[String]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala, 467, 467, 480)
something's wrong: no file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala in Array[String]RangePosition(file:///home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala, 274, 274, 287)
9Êúà 09, 2021 2:36:16 ÂçàÂâç org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
ÈáçÂ§ß: Internal error: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object Exp13
java.util.concurrent.CompletionException: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object Exp13
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:314)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:319)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1(CancelTokens.scala:42)
	at scala.meta.internal.metals.CancelTokens$.$anonfun$future$1$adapted(CancelTokens.scala:38)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: scala.reflect.internal.Symbols$CyclicReference: illegal cyclic reference involving object Exp13

[0m2021.09.09 02:40:37 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:40:49 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:40:57 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:41:31 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:41:31 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:41:31 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:41:55 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:42:31 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:43:39 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:43:39 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:43:39 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:44:28 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:44:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:44:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:44:42 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:48:59 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:48:59 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:48:59 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:50:08 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:50:08 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:50:26 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:51:08 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:51:59 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:51:59 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:51:59 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:53:50 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:54:57 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:54:57 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:55:09 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 02:55:09 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 03:48:43 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 03:48:43 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 03:49:01 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 03:49:01 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 03:51:33 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 03:51:33 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.09 03:51:59 INFO  shutting down Metals[0m
[0m2021.09.13 01:18:36 INFO  Started: Metals version 0.10.6 in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle' for client Visual Studio Code 1.60.0.[0m
[0m2021.09.13 01:18:37 INFO  time: initialize in 1.37s[0m
[0m2021.09.13 01:18:38 WARN  Build server is not auto-connectable.[0m
[0m2021.09.13 01:18:38 WARN  no build tool detected in workspace '/home/kuropikumin/Training/Scala/AlgorithmPazzle'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. [0m
[0m2021.09.13 01:18:38 WARN  no build target for: /home/kuropikumin/Training/Scala/AlgorithmPazzle/exp13/Exp13.scala[0m
[0m2021.09.13 01:18:40 INFO  no build target: using presentation compiler with only scala-library: 2.12.14[0m
